### PR TITLE
Made sure only anime marked shows are using this provider

### PR DESCRIPTION
**Pull request by p0psicles**

### CODE REVIEW

File: sickbeard/providers/anizb.py [hunk 1]
```diff
@@ -32,8 +32,9 @@ def __init__(self):
         # Provider Init
         NZBProvider.__init__(self, "Anizb")
 
-        # Credentials
         self.public = True
+        self.supports_absolute_numbering = True
+        self.anime_only = True
         self.confirmed = True
```
Comment (p0psicles) [inline sickbeard/providers/anizb.py]: Honestly I don't know what self.confirmed is used for. It got copied from another prov.

Comment (fernandog) [inline sickbeard/providers/anizb.py]: Its like "Trusted" torrents/users from KAT.

Comment (duramato) [inline sickbeard/providers/anizb.py]: confirmed its intended for verified torrents, ranked for internal torrents of the site

Comment (p0psicles) [inline sickbeard/providers/anizb.py]: Ok then i'll remove it

**Pull request merged**